src/stk/itk_bridge/itk_io.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <stk/image/volume.h>
+
+namespace stk {
+
+Volume read_itk_image(const std::string& file_name);
+
+void write_itk_image(const Volume& volume, const std::string& file_name);
```

Comment:
Maybe we can come up with better names for the functions than this...

Reply:
I guess these won't be used directly anyway?